packages/mock_server/ios/Flutter/Generated.xcconfig

```diff
@@ -0,0 +1,14 @@
+// This is a generated file; do not edit or check into version control.
```

Comment:
Could you please remove the generated files from version control? :)

Reply:
done